lib/katello/tasks/check_candlepin_content.rake

```diff
@@ -0,0 +1,16 @@
+namespace :katello do
+  desc "Check candlepin for missing repository content"
+  task :check_candlepin_content => ["environment", "check_ping"] do
+    logger = Logger.new(STDOUT)
+    User.current = User.anonymous_api_admin
+    repos = ::Katello::Repository.yum_type.in_default_view
+    bad_repos = repos.select { |repo| ::Katello::Util::CandlepinRepositoryChecker.repository_exist_in_backend?(repo) }
```

Comment:
Reject instead of select

```suggestion
    bad_repos = repos.reject { |repo| ::Katello::Util::CandlepinRepositoryChecker.repository_exist_in_backend?(repo) }
```